design/one-pager-resource-connectivity-mvp.md

```diff
@@ -410,10 +411,9 @@ could be implemented in one of two ways:
    order to determine that it is named `externalname` when submitting requests
    to the GCP API.
 
-This document suggests the first approach, both because it requires less logic
-and fewer Kubernetes API calls to implement, and because it enables Crossplane
+This document uses a combination of these approaches. For GCP case approach one is used both because it requires less logic and fewer Kubernetes API calls to implement, and because it enables Crossplane
 users to reference both external resources that are and are not modelled as
-Crossplane managed resources.
+Crossplane managed resources. For AWS case, a combination of both approaches are used.
```

Comment:
It feels like we should be consistent across providers with this.

Reply:
moved inter-resource references to #707